docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1292 @@
+preservation events, objects, agents, and rights. Enduro's supported
+preservation engines [a3m] and [Archivematica] can both parse valid PREMIS.xml
+files and add the contents to an AIP's [METS] file during preservation
+processing.
+
+When this activity is enabled, Enduro will check for the presence of a file
+named `premis.xml` in the `metadata` directory of a SIP. If a matching file
+is found in the expected location, Enduro will then run the [xmlvalidate]
+activity and validate the premis.xml file against the schema file found at
+the configured path.
+
+!!! note
+
+    For the PREMIS validation activity to succeed when enabled:
+
+    * The [libxml2](https://en.wikipedia.org/wiki/Libxml2) library must be
+      installed. Note that the project's Docker configuration automatically
+      installs this dependency in the Enduro Docker image.
+    * The file **MUST** be named `premis.xml`
+    * The file **MUST** be found in a `metadata` directory
+    * The metadata directory **must not** be nested inside any other directories
+    * The PREMIS schema file path must be accessible to Enduro
+
+**Default values**:
+
+```toml
+[validatePremis]
+enabled = true
+xsdPath = "/home/enduro/premis.xsd"
+```
+
+* `enabled`: Set to `true` by default. To skip PREMIS validation, change this
+  value to `false`.
+* `xsdPath`: Defines the filesystem path where Enduro can find the PREMIS 3
+  XML Schema Definition file to be used for validation.
+
+    Enduro does include a PREMIS 3.0 XSD file at installation, located at
+    `hack/xsd/premis.xsd` from the root Enduro installation directory.
+
+### Watched location configuration
+
+These configuration settings, when enabled, allow Enduro to initiate SIP ingest
+from a watched location. The configured watched location can either be a local
+filesystem directory or an object store bucket (such as one provided by MinIO,
+S3, or Azure). Once configured, any time a new zipped package is added to the
+location, Enduro's watcher (in this case, [Redis]) will see it and automatically
+initiate an ingest workflow. For more information, see:
+
+* [Initiate ingest via a watched location upload][watched-location]
+
+At this time, [Redis] is the only supported watcher - as such, the
+`redisAddress` and `redisList` fields are required.
+
+All other default parameters are S3-specific, and might not be needed for other
+watched location types. The default configuration included uses a [MinIO] bucket
+as the example watched location. MinIO uses Amazon [S3] syntax for its
+configuration properties. Different object stores may have different parameters
+to be configured. Consult the corresponding object store provider's
+documentation for more information.
+
+For **local filesystems**, the only other required field is a `url` parameter
+pointing to the target location - for example:
+
+```toml
+url = "file:///home/enduro/watchdir"
+```
+
+Note that when configuring a local filesystem directory as a watched location,
```

Comment:
The filesystem watcher also doesn't seem to use Redis at all, it's watching for inotify events or using https://github.com/radovskyb/watcher.